saturn-r/.env_deps

```diff
@@ -0,0 +1,3 @@
+VERSION=2021.02.22-1
```

Comment:
Note: I imagine these need to be changed to `2021.04.19` - let me know if so, I'll do it.

Reply:
Might as well - but `release-images` will also open PRs to do it after it runs (if needed).